srcpkgs/atom-bin/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'atom-bin'
+pkgname=atom-bin
+version=1.32.2
+revision=1
+depends="git gtk+ GConf libnotify libXtst nss python xdg-utils
+ desktop-file-utils alsa-lib libgnome-keyring libXScrnSaver"
+short_desc="Hackable text editor for the 21st century"
+conflicts="atom"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="https://atom.io/"
+only_for_archs="x86_64"
+nostrip=yes
+nocross=yes
```

Comment:
Also I think `nonfree` since we didn't do the build.

Reply:
So this means?

Reply:
I guess not `nonfree`, so no change.